tests/coding_sessions/test_routes_coding_sessions.py

```
@@ -0,0 +1,481 @@
+                           cookies={"taos_session": token}) as c:
+        resp = await c.get("/api/coding-sessions/cs-missing")
+        assert resp.status_code == 404
+    await app.state.coding_session_store.close()
+    await app.state.agent_registry.close()
+    await app.state.metrics.close()
+
+
+@pytest.mark.asyncio
+async def test_non_owner_gets_404(tmp_path):
+    app = create_app(data_dir=tmp_path / "data")
+    uid, token = await _make_client(app)
```

Ruff (0.15.18)

[warning] Unpacked variable `token` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)